src/components/AppNavigation/AppNavigation.vue

```diff
@@ -146,6 +146,7 @@ export default {
 	position: -webkit-sticky;
 	top: $header-height;
 	left: 0;
+	padding: 8px 12px;
```

Comment:
I would remove this padding altogether, so it's in line with e.g. the `ListItem` and how it's used in Mail.
```suggestion
```
And especially in horizontal direction, the padding uses a lot of space (nearly 10% of the available `AppNavigation`s horizontal space).